adapter.go

```diff
@@ -215,7 +221,13 @@ func savePolicyLine(ptype string, rule []string) *CasbinRule {
 
 // SavePolicy saves policy to database.
 func (a *Adapter) SavePolicy(model model.Model) error {
-	_, err := a.db.Model((*CasbinRule)(nil)).Where("id IS NOT NULL").Delete()
+	tx, err := a.db.Begin()
+	if err != nil {
+		return fmt.Errorf("start DB transaction: %v", err)
+	}
+	defer tx.Rollback()
```

Comment:
Should be `defer tx.Close()` .

see https://pkg.go.dev/github.com/go-pg/pg/v9?tab=doc#Tx.Close

Reply:
Oh... It's a new method in v9, cool